src/Resources/app/administration/src/module/sw-extension-store/component/sw-extension-store-statistics-promotion/sw-extension-store-statistics-promotion.html.twig

```diff
@@ -0,0 +1,18 @@
+<sw-card
+    v-if="showBanner"
+    class="sw-extension-store-statistics-promotion"
+>
+    <h3 class="sw-extension-store-statistics-promotion__title">
+        {{ $tc('title') }}
+    </h3>
+
+    <sw-button
+        class="sw-extension-store-statistics-promotion__cta"
+        variant="ghost"
+        style="background-color: white"
```

Comment:
`ghost` variant's CSS rules are overruling the CSS rules for this button.
I moved it to inline CSS for now to have this white background rule prioritized.

I don't want to lose time on it since it is still not the final design.
We will maybe need keep is like this or make it `!important` if it is in the final design QQ

Reply:
it's gone